test/v2/unit/coins/bch.js

```diff
@@ -21,28 +21,28 @@ describe('BCH:', function() {
 
     // we use mainnet bch so we can reuse the mainnet address examples
     it('should correctly convert addresses', function() {
```

Comment:
Can we leave one test case which ensures that the 'bech32' address type is still working for bch? Please add a comment as well explaining that this parameter is deprecated and that the test should be removed when the parameter is removed. Until then, it needs to keep working, so we should keep testing it.

Reply:
Added back all test cases (with comments to remove them) to ensure no regressions.